conjure/serverwriter.go

```diff
@@ -190,10 +190,10 @@ func astForHandlerMethodPathParams(methodBody *jen.Group, pathParams []*types.En
 }
 
 func astForHandlerMethodPathParam(methodBody *jen.Group, argDef *types.EndpointArgumentDefinition) {
-	strVar := transforms.SafeName(argDef.ParamID) + "Str"
```

Comment:
We can remove the `SafeName` function altogether now since it's not used anymore and since we're appending the `Arg` we shouldn't have collisions.